routing/turns_sound_settings.cpp

```diff
@@ -73,7 +72,23 @@ double Settings::ConvertUnitsToMeters(double distanceInUnits) const
       return MeasurementUtils::FeetToMeters(distanceInUnits);
   }
 
-  // m_lengthUnits is equal to LengthUnits::Undefined or to unknown value.
+  ASSERT(false, ());
+  return 0.;
+}
+
+double Settings::ConvertMetersToUnits(double distanceInMeters) const
+{
+  switch (m_lengthUnits)
+  {
+    case LengthUnits::Undefined:
```

Comment:
Do we really need this Undefined state in the code? If it is not used, it's better to stick to some default value (meters?) or always get it from our settings::